src/analyses/base.ml

```diff
@@ -9,13 +9,15 @@ module Q = Queries
 
 module GU = Goblintutil
 module ID = ValueDomain.ID
+module IDB = IntDomain.IntDomBranchingTuple
```

Comment:
Push into `invariant` since it's only used there.

Reply:
Done, in b6997e81485c9e7f97903ec78eb8450e96afa6c9.